README.md

```diff
@@ -5,3 +5,78 @@ emotions (happiness, surprise, fear, sadness, anger and disgust). For further de
 refer to "Emotionotopy in the Human Right Temporo-Parietal Cortex" paper (DOI: XXXX) and to
```

Comment:
While we are at this, shall we include a DOI? It should be https://doi.org/10.1038/s41467-019-13599-z

Reply:
Good call! I've pushed an update with the commit